src/net.cpp

```diff
@@ -2828,6 +2936,12 @@ void CNode::MarkReceivedMsgsForProcessing()
         // vRecvMsg contains only completed CNetMessage
         // the single possible partially deserialized message are held by TransportDeserializer
         nSizeAdded += msg.m_raw_message_size;
+
+        net_stats.Record(NetStats::Direction::RECV,
+                            ConnectedThroughNetwork(),
+                            m_conn_type,
+                            msg.m_type,
+                            msg.m_raw_message_size);
```

Comment:
Now that an effort has been made to remove the friendship between CConnman and CNode (see this PR: https://github.com/bitcoin/bitcoin/pull/27257) I'm not sure if this belongs here